### DISCUSSION TITLE
저희 리포지토리 포크안하고 쓸 수 있는 이유가 먼가여?!

### DISCUSSION THREAD

**Discussion by mdgarden** [EasyAndBeauty/DailyReceipt-FE-WEB#46] [Q&A]
organization이라서?!?

**Comment by IGhost-P** [comment]
맞습니다! organization이라서 포크를 하지 않고 사용을 할 수있니다!
저희 멤버가 아니라면 직접적으로 브랜치를 사용 할 수 없고, fork 이후 pull request에서 confirm 받는 형식으로 이루어 집니다!
